module/src/main/java/com/iluwatar/module/App.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * The MIT License Copyright (c) 2016 Amit Dixit
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package com.iluwatar.module;
+
+import java.io.FileNotFoundException;
+
+/**
+ * The Module pattern can be considered a Creational pattern and a Structural pattern. It manages
+ * the creation and organization of other elements, and groups them as the structural pattern does.
+ * An object that applies this pattern can provide the equivalent of a namespace, providing the
+ * initialization and finalization process of a static class or a class with static members with
+ * cleaner, more concise syntax and semantics.
+ * <p>
+ * The below example demonstrates a use case for testing two different modules: File Logger and
+ * Console Logger
+ * 
+ */
+public final class App {
+
+  public static FileLoggerModule fileLoggerModule = null;
+  public static ConsoleLoggerModule consoleLoggerModule = null;
```

Comment:
These are initialized to null automatically. Check for similar occurrences elsewhere.